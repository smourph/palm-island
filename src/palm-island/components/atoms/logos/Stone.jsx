import React from 'react';
import stone from '@internal/resources/images/logo-stone-small.png';
import './Logo.scss';

export const Stone = () => (
  <span className="logo logo-resource logo-stone">
    <img src={stone} alt="stone"/>
  </span>
);

export default Stone;
