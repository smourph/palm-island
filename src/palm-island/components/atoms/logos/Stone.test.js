import React from 'react';
import {render, screen} from '@testing-library/react';
import Stone from './Stone';

it('should render snapshot', () => {
  const {asFragment} = render(<Stone/>);
  expect(asFragment().firstChild).toMatchSnapshot();
});

it('should render stone', () => {
  render(<Stone/>);

  const icon = screen.getByAltText('stone');
  expect(icon).toBeInTheDocument();
  expect(icon).toBeVisible();
  expect(icon).toHaveAttribute('src', 'logo-stone-small.png');
});
