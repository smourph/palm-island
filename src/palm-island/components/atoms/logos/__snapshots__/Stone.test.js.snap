// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
<span
  class="logo logo-resource logo-stone"
>
  <img
    alt="stone"
    src="logo-stone-small.png"
  />
</span>
`;
